app/utils/wikiQuery.tsx

```diff
@@ -1,45 +1,20 @@
+import { IWikipediaContestantData } from './wikiFetch'
 
-import * as cheerio from 'cheerio'
-
-export const wikiUrl = "https://en.wikipedia.org/wiki/The_Amazing_Race_35"
-
-interface IWikipediaData {
-    parse: {
-        text: string
-    }
-}
-
-export interface IContestant {
+export interface ITeam {
```

Comment:
In the future, we should probably move these interfaces, including the two in `wikiFetch.tsx`, to a `models` or `interfaces` file, that way if anything changes they're a bit easier to track down. 

We are absolutely fine to leave this for now, but we may want to mark it as an issue if it's something I can't get to this week.

Reply:
Alright, I am opting to not do this right at the moment, because I just wanna keep development moving and I don't feel like this is a blocker, but I do agree with this premise so I created #22